ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -12,19 +14,14 @@
 
   let active: boolean = false;
 
-  enum TrackingEvent {
-    Track = "track",
-    Untrack = "untrack",
-  }
-
```

Comment:
Kind of unnecessary to move this to a new module with only this type when it is only used here...

Reply:
Consumers rely on the string value of these variants to catch the right events, but I didn't find a way to share that meaningfully. This was an attempt to do so.

Reply:
fwiw: I'm all for small modules with specific data :+1: Just because it's only used here now doesn't mean the tests of time will not force it to be used elsewhere or be added to.